utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/KafkaUniversalComponentTransformer.scala

```diff
@@ -62,7 +73,15 @@ abstract class KafkaUniversalComponentTransformer[T, TN <: TopicName: TopicValid
   protected def getTopicParam(
       implicit nodeId: NodeId
   ): WithError[ParameterCreatorWithNoDependency with ParameterExtractor[String]] = {
-    val topics = topicSelectionStrategy.getTopics(schemaRegistryClient)
+    val allTopics = getAllTopics
+    val topics = allTopics match {
```

Comment:
Are we sure that the topics will be only from our tenant namespace from hermes?

Reply:
Where did we do this check?

Reply:
After getting all topics from kafka and concatenating them with topics from schema registry I filter them using `topicSelectionStrategy`.

Now that I think about it should I move the filtering to `getAllTopics`?

Reply:
hm. I think it is a little bit different but still should work - filtering by tenant namespace is done here:
```scala
private def getTopicParam(topics: List[UnspecializedTopicName]) = {
  ParameterDeclaration
    .mandatory[String](topicParamName)
    .withCreator(
      modify = _.copy(editor =
        Some(
          FixedValuesParameterEditor(
            nullFixedValue +: topics
              .flatMap(topic => modelDependencies.namingStrategy.decodeName(topic.name)) // <--- here
              .sorted
              .map(v => FixedExpressionValue(s"'$v'", v))
          )
        )
      )
    )
}
```